src/main/java/dk/cs/aau/huppaal/presentations/HUPPAALPresentation.java

```diff
@@ -331,20 +318,154 @@ private void initializeColorSelector() {
     }
 
     private void initializeGenerateUppaalModelButton() {
-            final Color color = Color.GREY_BLUE;
-            final Color.Intensity colorIntensity = Color.Intensity.I800;
-
-            controller.generateUppaalModel.setMaskType(JFXRippler.RipplerMask.CIRCLE);
-            controller.generateUppaalModel.setRipplerFill(color.getTextColor(colorIntensity));
-        if(UPPAALDriverManager.getInstance() instanceof DummyUPPAALDriver){
+        var color = Color.GREY_BLUE;
+        var colorIntensity = Color.Intensity.I800;
+        controller.generateUppaalModel.setMaskType(JFXRippler.RipplerMask.CIRCLE);
+        controller.generateUppaalModel.setRipplerFill(color.getTextColor(colorIntensity));
+        if(UPPAALDriverManager.getInstance() instanceof DummyUPPAALDriver) {
             controller.generateUppaalModel.setEnabled(false);
             controller.generateUppaalModel.setOpacity(0.3);
         } else {
             controller.generateUppaalModel.setEnabled(true);
-            controller.generateUppaalModel.setOpacity(1);
+            controller.generateUppaalModel.setOpacity(1.0);
         }
     }
 
+    private void initializeRunConfigExecuteButton() {
+        var colorIntensity = Color.Intensity.I800;
+        var color = Color.GREY_BLUE;
+        controller.runConfigurationExecuteButton.setRipplerFill(color.getTextColor(colorIntensity));
+
+        JFXTooltip tooltip;
+        var c = controller.runConfigurationPicker.getSelectionModel().getSelectedItem();
+        if(c == null || c.runConfiguration().isEmpty()) {
+            tooltip = new JFXTooltip("no run configuration is selected");
+            controller.runConfigurationExecuteButton.setEnabled(false);
+            controller.runConfigurationExecuteButton.setOpacity(0.3);
+            controller.runConfigurationExecuteButton.setOnMouseClicked(e -> HUPPAAL.showToast("no run configuration is selected"));
```

Comment:
Capitalize 'N' in "no"

Reply:
Fixed in 4a4eaa25a8ec00e3b3fa4439a3e7078ea4d188fb